### DISCUSSION TITLE
i think i have a local ubuntu azerothcore server setup. Right now it is on localhost and i want to bind him to a static ip

### DISCUSSION THREAD

**Discussion by thermolab** [azerothcore/azerothcore-wotlk#16460] [Guides & Tips]
i have no idea how to do that
i have tried a couple of things but did not work

any help would be appreciated

**Comment by dewmguy** [comment]
This is done in the realmlist table of the server’s acore_realm database. https://www.azerothcore.org/wiki/realmlist#address

Address is set to localhost by default (aka 127.0.0.1) meaning you can only connect to the server if you are running the client on the same machine.

change this value to the server local IP (eg 192.168.1.123) to allow any computer on your local network to connect to the server.

change this value to your isp public IP address or domain name to allow anyone to connect to the server.

making the server public may require you to enable port forwarding rules in your network firewall or the firewall in Ubuntu (if you have one).
